packages/rancher-istio/1.12/rancher-istio/charts/Chart.yaml

```diff
@@ -11,14 +11,14 @@ annotations:
   catalog.cattle.io/requests-memory: 2314Mi
   catalog.cattle.io/type: cluster-tool
   catalog.cattle.io/ui-component: istio
-  catalog.cattle.io/upstream-version: 1.13.3
+  catalog.cattle.io/upstream-version: 1.12.6
 apiVersion: v1
-appVersion: 1.13.3
+appVersion: 1.12.6
 description: A basic Istio setup that installs with the istioctl. Refer to https://istio.io/latest/
   for details.
 icon: https://charts.rancher.io/assets/logos/istio.svg
 keywords:
 - networking
 - infrastructure
 name: rancher-istio
-version: 1.13.3
```

Comment:
this version shouldn't be modified, it's done in the package.yaml

Reply:
Added the appropriate version.